lib/tty/tty-slang.c

```diff
@@ -723,6 +733,23 @@ tty_tigetstr (const char *terminfo_cap, const char *termcap_cap)
 
 /* --------------------------------------------------------------------------------------------- */
 
+// Warning: S-Lang doesn't support more than two parameters
+char *
+tty_tiparm (const char *str, ...)
+{
+    va_list args;
+    int p1, p2;
+
+    va_start (args, str);
+    p1 = va_arg (args, int);
+    p2 = va_arg (args, int);
```

Comment:
pulling a fixed number of arguments in a formally varargs function sounds like a recipe for disaster.